operators/README.md

```diff
@@ -71,10 +78,22 @@ The Instance Operator implements the backend logic necessary to spawn new enviro
 - **Instance** defines an instance of a certain template. The manipulation of those objects triggers the reconciliation logic in the operator, which creates/destroy associated resources (e.g.; Virtual Machines).
 
 ### Persistent Feature
-The Instance Operator allows you to create persistent Virtual Machines. Persistent means that the VM can be stopped and restarted, deleted and recreated without data loss. To make this possible, the content of the VM's disk is stored in a PVC.
+
+The Instance Operator allows you to create persistent VirtualMachines (VM). Persistent means that the VM can be stopped and restarted, deleted and recreated without data loss. 
 In order to use this feature, [the CDI operator](../infrastructure/virtualization/README.md) has to be installed.
-Granted the CDI operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which version to create is performed looking at the corresponding field in the `template` spec.
-N.B. The process of creating a persistent VirtualMachine can take a bit more time with the respect to a normal one (5-10 mins, depending on the size of the image). However when you recreate the VM you will not have to wait such that time.
+
+Provided that the CDI Operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which kind of VM is wanted, i.e. persistent or not, is performed looking at the corresponding field in the `template` spec. In the same way you can choose the size of the persistent disk that the VM will have.
+
+It has been decided to use a VM instead of directly creating a VirtualMachine Instance (VMI) because the start and stop process is already managed by KubeVirt. In this way the VMI is easily stopped and started again modifying the running field of Instance resource (this can be easily done from the user interface).
```

Comment:
```suggestion
Focusing on the resources created by the controller, it is worth highlighting that in the persistent case the Instance Operator instantiates a KubeVirt VirtualMachine object instead of a simple VirtualMachineInstance one. Indeed, this design choice allows to delegate the start and stop process of a VM to KubeVirt itself. Hence, a persistent VM can be easily stopped and started again modifying the `running` field of the Instance resource (optionally through the graphical user interface).
```